docs/low-level-api.md

```diff
@@ -11,30 +11,24 @@ import { actionToPath, pathToAction } from 'redux-first-router'
 const { routesMap } = store.getState().location
 
 const path = actionToPath(action, routesMap, querySerializer)
-const action = pathToAction(path, routesMap, querySerializer, basename)
+const action = pathToAction(path, routesMap, querySerializer, basename, strict)
 ```
 
-The `querySerializer` argument is optional and the same as the one passed to `connectRoutes`.
+The `querySerializer`, `basename`, and `strict` arguments are optionals.
```

Comment:
"optional"

Reply:
I should have been more focus in English class, sorry (adjectives should agree with their nouns in french). :(